ui/app/src/components/shared/ResourceCard.tsx

```diff
@@ -273,6 +294,18 @@ export const ResourceCard: React.FunctionComponent<ResourceCardProps> = (
                     {props.resource.user.name}
                   </Stack.Item>
                 </Stack>
+                {props.resource.resourceType === ResourceType.UserResource &&
+                  (props.resource as UserResource).ownerId &&
+                  (props.resource as UserResource).ownerId.trim() && (
+                    <Stack horizontal tokens={{ childrenGap: 5 }}>
+                      <Stack.Item style={calloutKeyStyles}>
+                        Owner ID:
+                      </Stack.Item>
+                      <Stack.Item style={calloutValueStyles}>
+                        {(props.resource as UserResource).ownerId}
```

Comment:
Can we also include the users email, if available and provide it as a mailto hyperlink that can be clicked on.